Jenkinsfiles/Jenkinsfile.build_ami

```diff
@@ -126,12 +126,13 @@ pipeline {
     stage('Set private key file') {
       steps {
         script {
+          if (params.ENV == "prod") {
```

Comment:
If there's a way to do it without the conditionals that would be great, but LGTM